src/App.js

```diff
@@ -1,26 +1,28 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import data from './api/movies.json';
 
-export class App extends Component {
-  state = {
-    movies: data,
-  };
+export const App = () => {
+  const [movies, setMovies] = useState(data);
+
+  const addMovie = (movie) => {
+    const findedDuplicate = movies.some(film => film.imdbId === movie.imdbId);
```

Comment:
better to make a variable 'isMovieUnique' and setMovie if isMovieUnique === true. 
It's always easier to read positive conditions than negative ones.